app/uk/gov/hmrc/servicedependencies/service/DeadLetterHandler.scala

```diff
@@ -60,12 +59,14 @@ class DeadLetterHandler @Inject()(
     }.get
 
   if (config.isEnabled) {
-    SqsSource(queueUrl, settings)(awsSqsClient)
-      .mapAsync(10)(processMessage)
-      .runWith(SqsAckSink(queueUrl)(awsSqsClient))
-      .recoverWith {
-        case NonFatal(e) => logger.error(e.getMessage, e); Future.failed(e)
-      }
+    config.sqsDeadLetterQueues.foreach { queueUrl =>
+      SqsSource(queueUrl, settings)(awsSqsClient)
+        .mapAsync(10)(processMessage)
+        .runWith(SqsAckSink(queueUrl)(awsSqsClient))
+        .recoverWith {
+          case NonFatal(e) => logger.error(e.getMessage, e); Future.failed(e)
```

Comment:
will this message include the name of the deadletter queue, or should be add `$queueUrl` to the message?